daemon/networkdriver/portallocator/portallocator.go

```diff
@@ -59,6 +68,29 @@ func NewErrPortAlreadyAllocated(ip string, port int) ErrPortAlreadyAllocated {
 	}
 }
 
+func init() {
+	const param = "/proc/sys/net/ipv4/ip_local_port_range"
+
+	if line, err := ioutil.ReadFile(param); err != nil {
+		log.Errorf("Failed to read %s kernel parameter: %s", param, err.Error())
+	} else {
```

Comment:
I'm little confused by flow with elses.
I'm prefer if you'd use like:
```
f, err := os.open(param)
if err != nil {
    log.Errorf(...)
    return
}
```
and so on